EventListener/TulipAPIQueueSubscriber.php

```diff
@@ -22,13 +23,22 @@ class TulipAPIQueueSubscriber implements EventSubscriber
     private $queueManager;
 
     /**
+     * The boolean indicating that the kernel is in debug mode and queued objects are sent directly.
+     *
+     * @var bool
+     */
+    private $debug;
+
+    /**
      * Constructs a new TulipAPIQueueSubscriber instance.
      *
      * @param QueueManager $queueManager
+     * @param bool         $debug
      */
-    public function __construct(QueueManager $queueManager)
+    public function __construct(QueueManager $queueManager, $debug = false)
     {
         $this->queueManager = $queueManager;
+        $this->debug = true;
```

Comment:
Looks like a bug to me. Someone should check the PR instead of self-pulling, unless required due to some kind of deadline and / or lack of people that can check.